src/sentry/static/sentry/app/views/alerts/list/index.tsx

```diff
@@ -60,11 +61,65 @@ class IncidentsList extends AsyncComponent<Props, State & AsyncComponent['state'
       ],
     ];
   }
+  async onLoadAllEndpointsSuccess() {
+    const {incidentList} = this.state;
+    if (incidentList.length === 0) {
+      // Check if they have rules or not, to know which empty state message to display
+      const {params} = this.props;
+
+      this.setState({
+        loading: true,
+      });
+
+      try {
+        const alert_rules = await this.api.requestPromise(
```

Comment:
Is it possible to have this enforced, so I'm reminded before committing? :-)

Reply:
It is possible, but it can be annoying when working with APIs that use underscores, I can see if there are any options that would apply to our use case though.